index.js

```diff
@@ -1,22 +1,23 @@
 const {
-  GraphQLFloat,
-  GraphQLInt,
-  GraphQLString,
   GraphQLNonNull,
-  isNonNullType,
-  isScalarType,
   GraphQLList,
-  isListType
+  TypeInfo,
+  ValidationContext,
+  visit,
+  visitWithTypeInfo,
+  separateOperations,
+  GraphQLError
 } = require('graphql')
+const QueryValidationVisitor = require('./lib/QueryValidationVisitor.js')
 const { getDirective, mapSchema, MapperKind } = require('@graphql-tools/utils')
-const ConstraintStringType = require('./scalars/string')
-const ConstraintNumberType = require('./scalars/number')
+const { getConstraintTypeObject, getScalarType } = require('./lib/typeutils')
+const { constraintDirectiveTypeDefs } = require('./lib/typeDefs')
```

Comment:
`type-defs.js`, let's avoid upper casing in file names where possible. I've been bitten by this too many times in the past between case sensitive vs non-case sensitive file systems 😅